src/frontends/tensorflow_common/src/op/expand_dims.cpp

```diff
@@ -14,9 +19,31 @@ namespace tensorflow {
 namespace op {
 
 OutputVector translate_expand_dims_op(const NodeContext& node) {
-    default_op_checks(node, 2, {"ExpandDims", "EXPAND_DIMS"});
+    default_op_checks(node, 2, {"ExpandDims", "EXPAND_DIMS"}, true);
     auto input = node.get_input(0);
     auto axis = node.get_input(1);
+    auto complex_type_mark = as_type_ptr<ComplexTypeMark>(input.get_node_shared_ptr());
+
+    if (complex_type_mark) {
+        element::Type complex_part_type = complex_type_mark->get_complex_part_type();
+        input = complex_type_mark->input_value(0);
+
+        auto const_zero = make_shared<v0::Constant>(element::i32, Shape{1}, 0);
```

Comment:
`axis` can also be of `int64` type. Please use `utils` rountine for creating scalar as `axis` type. You should know of it:)

Reply:
https://www.tensorflow.org/api_docs/python/tf/raw_ops/ExpandDims

Reply:
Oops, sorry for that! I looked on the unit test in the code base and it uses only `int32`.

I'll fix that right away.